packages/slice-machine/src/modules/environment/types.ts

```diff
@@ -1,9 +1,3 @@
 import { FrontEndEnvironment } from "@lib/models/common/Environment";
-import Warning from "@models/common/Warning";
-import { ConfigErrors } from "@models/server/ServerState";
 
-export type EnvironmentStoreType = {
-  env: FrontEndEnvironment;
-  warnings: ReadonlyArray<Warning>;
-  configErrors: ConfigErrors;
-};
+export type EnvironmentStoreType = FrontEndEnvironment;
```

Comment:
Was it for the naming coherence or to not rename everythign that you decided to keep that ?

Reply:
Just for keeping the name convention for store type